e2e/e2etests/test_message_passing_zevm.go

```diff
@@ -0,0 +1,332 @@
+	}
+	r.Logger.Info(fmt.Sprintf("🔄 Cctx mined for contract call chain zevm %s", cctx.Index))
+
+	// On finalization the Fungible module calls the onReceive function which in turn calls the onZetaMessage function on the destination contract
+	receipt, err = r.ZEVMClient.TransactionReceipt(r.Ctx, ethcommon.HexToHash(cctx.GetCurrentOutTxParam().OutboundTxHash))
+	if err != nil {
+		panic(err)
+	}
+	if receipt.Status != 1 {
+		panic("tx failed")
+	}
+	testDAppZEVM, err := testdapp.NewTestDApp(r.ZevmTestDAppAddr, r.ZEVMClient)
+	if err != nil {
+		panic(err)
+	}
+	receivedHelloWorldEvent := false
+	for _, log := range receipt.Logs {
+		_, err := testDAppZEVM.ParseHelloWorldEvent(*log)
+		if err == nil {
+			r.Logger.Info("Received HelloWorld event")
+			receivedHelloWorldEvent = true
+		}
+	}
+	if !receivedHelloWorldEvent {
+		panic("expected HelloWorld event")
+	}
+}
+
+func TestMessagePassingEVMtoZEVMRevert(r *runner.E2ERunner, args []string) {
+
+	if len(args) != 1 {
+		panic("TestMessagePassingRevert requires exactly one argument for the amount.")
+	}
+
+	amount, ok := big.NewInt(0).SetString(args[0], 10)
+	if !ok {
+		panic("Invalid amount specified for TestMessagePassingRevert.")
+	}
+
+	// Set destination details
+	zEVMChainID, err := r.ZEVMClient.ChainID(r.Ctx)
+	if err != nil {
+		panic(err)
+	}
+	destinationAddress := r.ZevmTestDAppAddr
+
+	// Contract call originates from EVM chain
+	auth := r.EVMAuth
+	tx, err := r.ZetaEth.Approve(auth, r.EvmTestDAppAddr, amount)
+	if err != nil {
+		panic(err)
+	}
+	r.Logger.Info("Approve tx hash: %s", tx.Hash().Hex())
+	receipt := utils.MustWaitForTxReceipt(r.Ctx, r.EVMClient, tx, r.Logger, r.ReceiptTimeout)
+	if receipt.Status != 1 {
+		panic("tx failed")
+	}
+	r.Logger.Info("Approve tx receipt: %d", receipt.Status)
+	testDAppEVM, err := testdapp.NewTestDApp(r.EvmTestDAppAddr, r.EVMClient)
+	if err != nil {
+		panic(err)
+	}
+	// Call the SendHelloWorld function on the EVM dapp Contract which would in turn create a new send, to be picked up by the zeta-clients
+	// set Do revert to true which adds a message to signal the ZEVM zetaReceiver to revert the transaction
+	tx, err = testDAppEVM.SendHelloWorld(auth, destinationAddress, zEVMChainID, amount, true)
+	if err != nil {
+		panic(err)
+	}
+	r.Logger.Info("TestDApp.SendHello tx hash: %s", tx.Hash().Hex())
+	receipt = utils.MustWaitForTxReceipt(r.Ctx, r.EVMClient, tx, r.Logger, r.ReceiptTimeout)
+
+	// New inbound message picked up by zeta-clients and voted on by observers to initiate a contract call on zEVM which would revert the transaction
+	// A revert transaction is created and gets fialized on the original sender chain.
+	cctx := utils.WaitCctxMinedByInTxHash(r.Ctx, receipt.TxHash.String(), r.CctxClient, r.Logger, r.CctxTimeout)
+	if cctx.CctxStatus.Status != cctxtypes.CctxStatus_Reverted {
+		panic("expected cctx to be reverted")
+	}
+	r.Logger.Print(fmt.Sprintf("🔄 Cctx mined for revert contract call chain zevm %s", cctx.Index))
```

Comment:
```suggestion
	r.Logger.Info(fmt.Sprintf("🔄 Cctx mined for revert contract call chain zevm %s", cctx.Index))
```
To keep default logs minimal

Reply:
d895bb494c3f8b659de6508e4307d4770c6d05f8